test/ci/macos/run-ngraph-tf-mac-build.sh

```diff
@@ -223,7 +223,7 @@ if [ -z "${tf_dir}" ] ; then
     # If installing into the OS, use:
     # sudo --preserve-env --set-home pip install --ignore-installed ${PIP_INSTALL_EXTRA_ARGS:-} "${WHEEL_FILE}"
     # Here we are installing into a virtual environment, so DO NOT USE SUDO!!!
-    pip install -U tensorflow
+    pip install -U tensorflow==1.12.0rc1
```

Comment:
This should be `1.12.0rc2` as the rc2 is now out (a few days ago).

Reply:
OK, updated to 1.12.0rc2